www/templates/html/scripts/frontend.js

```diff
@@ -0,0 +1,13 @@
+require(['jquery'], function($){
+	WDN.initializePlugin('modal', [function() {
```

Comment:
Good idea! I took this straight from the examples without much thought. I will see if I can streamline it.